core/ledger/ledgerstorage/store_test.go

```diff
@@ -34,6 +36,24 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+func couchDBSetup() (addr string, cleanup func()) {
```

Comment:
what does couchDB have to do with the abstract ledgerstorage?

Reply:
I remove it